src/common/MemberSelector/MemberSelector.tsx

```diff
@@ -70,18 +75,25 @@ const MemberSelector = ({ value, onValueChange , loginUser }: SelectFieldProps)
       <SelectTrigger
         data-testid="memberId"
         className={`border rounded-md p-2 border-gray-300`}
-      >
-        <SelectValue data-testId="select" placeholder="Select user" />
+      >{roleBaseMembers.length ? (
```

Comment:
This condition seems odd. What happens if there are no members. User would always see a loader which is a wrong UX.

Reply:
i have fixed this using `useSWR`